hazelcast-jet-core/src/main/java/com/hazelcast/jet/DAG.java

```diff
@@ -156,12 +156,14 @@ public DAG edge(Edge edge) {
         if (getInboundEdges(edge.getDestName())
                 .stream().anyMatch(e -> e.getDestOrdinal() == edge.getDestOrdinal())) {
             throw new IllegalArgumentException("Vertex '" + edge.getDestName()
-                    + "' already has an inbound edge at ordinal " + edge.getDestOrdinal());
+                    + "' already has an inbound edge at ordinal " + edge.getDestOrdinal()
+                    + ", use Edge.from().to() to specify an ordinal");
```

Comment:
It's good for beginners, as I myself first didn't know, what to do. I'll try to elaborate.